bot/seasons/halloween/hacktoberstats.py

```diff
@@ -264,6 +271,7 @@ def build_embed(self, github_username: str, prs: List[dict]) -> discord.Embed:
                         ),
                     }
                     outlist.append(itemdict)
+                logging.info(f"Found {len(outlist)} valid Hacktoberfest PRs for GitHub user: '{github_username}'")
```

Comment:
The logging from here will be the same as the one for total issues most of the time and imo the command has a lot of logging now.
Have you considered merging those two to a single log entry?
`f"Found {len(jsonresp['items'])} total and {len(outlist)} valid Hacktoberfest PRs for GitHub user: '{github_username}'"`

Reply:
That's a good point, I'll batch this change with any review changes.